docs/src/persistent_tasks.md

```diff
@@ -32,6 +32,28 @@ fails to precompile: `using PkgA` runs `PkgA.__init__()`, which
 leaves the `Timer` `Task` running, and that causes precompilation
 of `PkgB` to hang.
 
+## Example with `expr`
+
+You can test that an expression using your package finishes without leaving any persistent
+tasks by passing a quoted expression:
+
+```julia
+Aqua.test_persistent_tasks(MyPackage, quote
+    # Code to run after loading MyPackage
+    server = MyPackage.start_server()
+    MyPackage.stop_server!(server)
+end)
+```
+
+here is an example test with a dummy expr which will obviously fail, because it's explicitly
+spawning a Task that never dies!
+```julia
+# This will fail because of the dangling spawn.
```

Comment:
```suggestion
Here is an example test with a dummy expr which will obviously fail, because it's explicitly
spawning a Task that never dies.
```@repl
```
some reformulation, adding the error message of the failure